games/sprig_dodge.js

```diff
@@ -0,0 +1,180 @@
+/*
+@title: sprig_dodging
+@author: sam liu and lucas
+
+Instructions:
+
+Welcome to Sprig!!!
+
+Hit "run" to execute the code and
+start the game (you can also press shift+enter).
+
+To beat each level you'll have to edit the code.
+
+The code for this game starts below this comment.
+
+Made based on this workshop https://workshops.hackclub.com/sprig_dodge/
+
+The goal is to dodge the fireballs.
+
+Click the "open help" to discover your toolkit.
+
+--------
+Level 1
+--------
+
+Add the player and obstacles in map
+
+--------
+Level 2
+--------
+
+Add controls to right, use "d" as input
+
+--------
+Level 3
+--------
+
+Fix the code!
+This cause this error:
+Uncaught TypeError: Cannot read properties of undefined (reading 'length') on line 153 in column 33. Open the browser console for more information.
```

Comment:
```suggestion
Uncaught TypeError: Cannot read properties of undefined (reading 'length') on line ... in column 33. Open the browser console for more information.
```
The line probably won't be the 153 after the previous levels